### DISCUSSION TITLE
What is difference between React and Redux?

### DISCUSSION THREAD

**Discussion by ra0321** [community/community#36999] [Other Feature Feedback, Questions, & Ideas] [answered]
What is difference between React and Redux?

**Comment by jupiterwebdev** [comment, marked as answer]
Redux can be used with React. both are independent of each other. Redux is a state managing library used in JavaScript apps. It simply manages the state of your application or in other words, it is used to manage the data of the application.

**Comment by davidboxler** [comment]
React and Redux are complementary tools that can be used together in one application. React focuses on building dynamic user interfaces, while Redux focuses on managing global application state.
